wordpress-mobile/WordPress-Android
# Issue/8642 sites quick start

### Fix
Update the ***Quick Start*** list item in the ***Sites*** tab as described in https://github.com/wordpress-mobile/WordPress-Android/issues/8642.  See the screenshots below for illustration.

![v2_sites_progress](https://user-images.githubusercontent.com/3827611/49296094-f5195400-f47c-11e8-854f-7a3554958e0c.png)

#### Note
There are two `// TODO: Quick Start -` items included in these changes that are related to new tasks. They will be addressed in a separate pull request as described in https://github.com/wordpress-mobile/WordPress-Android/issues/8647.  The more menu in the ***Next Steps*** view does nothing with these changes.  It will be addressed in a separate pull request as described in https://github.com/wordpress-mobile/WordPress-Android/issues/8665.

### Test
0. Select site with Quick Start in progress.
1. Go to ***Sites*** tab.
2. Notice ***Next Steps*** view is shown.
3. Notice ***Customize Your Site*** and ***Grow Your Audience*** are enabled.
4. Tap ***Customize Your Site*** item.
5. Notice ***Next Steps*** list is shown.
6. Complete ***Customize Your Site*** tasks.
7. Go to ***Sites*** tab.
8. Notice ***Customize Your Site*** is disabled and ***Grow Your Audience*** is enabled.
9. Tap ***Grow Your Audience*** item.
10. Notice ***Next Steps*** list is shown.
11. Complete ***Grow Your Audience*** tasks.
12. Go to ***Sites*** tab.
13. Notice ***Customize Your Site*** and ***Grow Your Audience*** are disabled.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.
